include/internal/rand.h

```diff
@@ -15,14 +15,22 @@
 
 /*
  * Default security strength (in the sense of [NIST SP 800-90Ar1])
- * of the default OpenSSL DRBG, and the corresponding NID.
  *
- * Currently supported values: 128, 192, 256
+ * NIST SP 800-90Ar1 supports the strength of the DRBG being smaller than that
+ * of the cipher by collecting less entropy. The current DRBG implemantion does
+ * not take RAND_DRBG_STRENGTH into account and sets the strength of the DRBG
+ * to that of the cipher.
```

Comment:
This is not true. `drbg->strength` is passed as third argument to to the `get_entropy` callback which implies that a minimum amount of `RAND_DRBG_STRENGTH` entropy bits is requested for (re-)seeding.

Reply:
But drbg->strength is not based on RAND_DRBG_STRENGTH but on RAND_DRBG_NID, it's keylen * 8. This is not required by NIST SP 800-90. With AES-256-CTR you should be able to set it to 128, 192 or 256, but the code currently doesn't allow this.

Reply:
Oh sorry, you're right. It's ok if you leave the comment here. But please fix the typo: implemantion -> implementation.